metrics/mixin/alerts/mirroring.libsonnet

```diff
@@ -23,7 +23,7 @@
           {
             alert: 'OdfPoolMirroringImageHealth',
             expr: |||
-              (ocs_pool_mirroring_image_health{%(ocsExporterSelector)s}  * on (namespace) group_left() (max by(namespace) (ocs_pool_mirroring_status{%(ocsExporterSelector)s}))) == 1
+              (ocs_pool_mirroring_image_health{%(ocsExporterSelector)s}  * on (namespace, name) group_left() (ocs_pool_mirroring_status{%(ocsExporterSelector)s})) == 1
```

Comment:
OdfPoolMirroringImageHealth alert will be triggered only when ocs_pool_mirroring_status is "1" (enabled) for the corresponding pool.